packages/build/src/run-draft.spec.ts

```diff
@@ -0,0 +1,190 @@
+import chai, { expect } from 'chai';
+import path from 'path';
+import sinon from 'ts-sinon';
+import { ALL_BUILD_VARIANTS, Config } from './config';
+import { uploadArtifactToDownloadCenter as uploadArtifactToDownloadCenterFn } from './download-center';
+import { downloadArtifactFromEvergreen as downloadArtifactFromEvergreenFn } from './evergreen';
+import { generateChangelog as generateChangelogFn } from './git';
+import { GithubRepo } from './github-repo';
+import { ensureGithubReleaseExistsAndUpdateChangelogFn, runDraft } from './run-draft';
+import { PackageInformation } from './tarball';
+
+chai.use(require('sinon-chai'));
+
+function createStubRepo(overrides?: any): GithubRepo {
+  return sinon.createStubInstance(GithubRepo, overrides) as unknown as GithubRepo;
+}
+
+describe('draft', () => {
+  let config: Config;
+  let githubRepo: GithubRepo;
+  let uploadArtifactToDownloadCenter: typeof uploadArtifactToDownloadCenterFn;
+  let downloadArtifactFromEvergreen: typeof downloadArtifactFromEvergreenFn;
+
+  beforeEach(() => {
+    config = {
+      version: 'version',
+      appleNotarizationBundleId: 'appleNotarizationBundleId',
+      input: 'input',
+      execInput: 'execInput',
+      executablePath: 'executablePath',
+      mongocryptdPath: 'mongocryptdPath',
+      outputDir: 'outputDir',
+      analyticsConfigFilePath: 'analyticsConfigFilePath',
+      project: 'project',
+      revision: 'revision',
+      branch: 'branch',
+      evgAwsKey: 'evgAwsKey',
+      evgAwsSecret: 'evgAwsSecret',
+      downloadCenterAwsKey: 'downloadCenterAwsKey',
+      downloadCenterAwsSecret: 'downloadCenterAwsSecret',
+      githubToken: 'githubToken',
+      segmentKey: 'segmentKey',
+      appleNotarizationUsername: 'appleNotarizationUsername',
+      appleNotarizationApplicationPassword: 'appleNotarizationApplicationPassword',
+      appleCodesignIdentity: 'appleCodesignIdentity',
+      isCi: true,
+      platform: 'platform',
+      repo: {
+        owner: 'owner',
+        repo: 'repo',
+      },
+      packageInformation: {
+        metadata: {
+          name: 'mongosh',
+          version: 'packageVersion',
+          description: 'The best shell you ever had.',
+          homepage: 'https://mongodb.com',
+          maintainer: 'We, us, everyone.'
+        }
+      } as PackageInformation,
+      execNodeVersion: process.version,
+      rootDir: path.resolve(__dirname, '..', '..')
+    };
+
+    uploadArtifactToDownloadCenter = sinon.spy();
+    downloadArtifactFromEvergreen = sinon.spy();
+  });
+
+  describe('runDraft', () => {
+    let ensureGithubReleaseExistsAndUpdateChangelog: typeof ensureGithubReleaseExistsAndUpdateChangelogFn;
+
+    beforeEach(() => {
+      ensureGithubReleaseExistsAndUpdateChangelog = sinon.stub();
+    });
+
+    it('works as expected', async() => {
```

Comment:
just a super silly thing, maybe this could be broken in a way that who reads understand what is the expected behavior behind 'works as expected', something like:

```
context('when there is a triggeringGitTag', () => {
   beforeEach(() => runDraft(...));
   
   it ('generates a changelog', () => {
      expect(ensureGithubReleaseExistsAndUpdateChangelog).to.have.been.calledWith(
        config.version, `v${config.version}`, githubRepo
      );
   })
   
   /// ....
})
```

Reply:
Yeah good point, that makes it a lot more explicit and readable, like that.